backend/apps/investors/models.py

```diff
@@ -42,13 +42,18 @@
 
 
 class InvestorProfile(TimeStampedModel):
-    user = models.ForeignKey(User, on_delete=models.CASCADE,
-                             related_name='investor_profile')
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='investor_profile'
+    )
     company_name = models.CharField(max_length=200)
     full_name = models.CharField(max_length=200)
-    description = models.TextField()
-    investment_range_min = models.DecimalField(max_digits=12, decimal_places=2)
-    investment_range_max = models.DecimalField(max_digits=12, decimal_places=2)
+    description = models.TextField(blank=True, default="")
+    investment_range_min = models.DecimalField(
+        max_digits=12, decimal_places=2, null=True, blank=True
+    )
+    investment_range_max = models.DecimalField(
+        max_digits=12, decimal_places=2, null=True, blank=True
+    )
```

Comment:
`investment_range_min` and `investment_range_max` have been changed to null=True, blank=True. The `clean()` method performs a direct comparison and will raise TypeError when one or both are None. Guard the comparison for None:
```
def clean(self):
    if self.investment_range_min is not None and self.investment_range_max is not None:
        if self.investment_range_max < self.investment_range_min:
            raise ValidationError("investment_range_max must be >= investment_range_min")
```